bokehjs/src/lib/models/glyphs/image.ts

```diff
@@ -13,20 +14,33 @@ export class ImageView extends ImageBaseView {
   declare model: Image
   declare visuals: Image.Visuals
 
+  /** @internal */
+  declare glglyph?: ImageGL
+
+  override async load_glglyph() {
+    const {ImageGL} = await import("./webgl/image")
+    return ImageGL
+  }
+
   override connect_signals(): void {
     super.connect_signals()
     this.connect(this.model.color_mapper.change, () => this._update_image())
   }
 
   protected _update_image(): void {
+    if (this.glglyph != null) {
+      this.glglyph.set_image_changed()
+    }
+
     // Only reset image_data if already initialized
     if (this._image_data != null) {
       this._set_data(null)
       this.renderer.request_render()
     }
   }
 
-  protected _flat_img_to_buf8(img: NDArrayType<number>): Uint8ClampedArray {
+  // public so can be called by ImageGL class.
+  public _flat_img_to_buf8(img: NDArrayType<number>): Uint8ClampedArray {
```

Comment:
Any reason not to drop the leading underscore, if this needs to be public? Then the comment is also probably not strictly needed either. Same comment in other locations as well.

Reply:
Actually this can revert to being `protected`. In the end, rather than calling this function from the `ImageGL` class I am l letting the non-WebGL glyph classes call it is their usual manner, which is consistent with how other WebGL glyphs are implemented.

Done in `8362165`.

Reply:
@ianthomas23 just FYI if you post the commit hash w/o the backtick quotes then GH will automatically generate a link to the commit iniline

Reply:
Oh yeah, so it does!